### PR TITLE
Adds remote docker / docker compose support

### PR DESCRIPTION
**Pull request by lunamidori5**
This PR lets users install the plug in a sandbox / docker based image for easier installing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,13 +6,19 @@
...
 1. Check if your `Node.js` version is >= **14**.
 2. Run `npm install` to install the dependencies.
 3. replace your GEMINI api key in `.env.template` and rename the file  to `.env`
 <!-- 3. you need to either be running a local model on `http://localhost:1234` or change the openAI config in `src/utils.js` -->
 
+## Installing via Docker
+
+1. Clone the repo
+2. Run `docker compose up -d --build` to install the plugin.
+3. The plugin is now running on ``**yourip**:3000``
```
Comment (lunamidori5) [inline README.md:20]: Local host is not used here. Instead, we employ the host computer's IP address. This approach simplifies server access for the end user.

**Comment by aj47** [comment]
Awesome @lunamidori5 !

Have you tested it to be working? 
I'm just confused at the flow;
- I don't see the gateway ai process being ran in the dockerfile
- After the docker compose is ran per instructions, what are the next steps? having a server running on :3000 seems like it's not enough to get the chrome extension loaded

**Comment by lunamidori5** [comment]
I followed the read me, if its not in the read me, its not in the dockerfile, after this is merged I can update it with with the needed installs for gateway if that is added to the readme

**Comment by lunamidori5** [comment]
Sorry @aj47

**Comment by aj47** [comment]
It's in the readme, just not in the install section cos it can just be run with npx . Anyway I don't see how you can even run and use this still

**Comment by lunamidori5** [comment]
@aj47 its in the updated readme, ``docker compose up -d --build`` will build it, if you would like once this is merged I can fix it to install `gateway` if you wish

**Pull request closed without merging**